packages/portfolio-deploy/src/chain-info.build.js

```diff
@@ -89,14 +93,23 @@ const parsePeers = strs => {
  *                 .withOpts({rpcAddrs: ['https...]});
  *   const info = await agd.query(['bank', 'balances', 'agoric1...]);
  * @param {{ execFileSync: typeof import('child_process').execFileSync}} io
+ * @param {{ podName?: string, container?: string }} [options] - Optional configuration for kubectl
  */
-const makeAgd = ({ execFileSync }) => {
+const makeAgd = (
+  { execFileSync },
+  { podName = 'agoriclocal-genesis-0', container = 'validator' } = {},
+) => {
   const exec = (
     /** @type {string[]} */
     args,
     /** @type {import('node:child_process').ExecFileSyncOptionsWithStringEncoding} */
     opts = { encoding: 'utf-8', stdio: ['ignore', 'pipe', 'pipe'] },
-  ) => execFileSync('agd', [...args], opts);
+  ) =>
+    execFileSync(
+      'kubectl',
+      ['exec', podName, '-c', container, '--', 'agd', ...args],
+      opts,
+    );
```

Comment:
With the change I mentioned above, this would be something like:
```suggestion
  ) =>
    execFileSync(
      appCmd,
      [...appCmdArgs, ...args],
      opts,
    );
```